### PR TITLE
#104 - Creates Access Safe Component

### PR DESCRIPTION
**Pull request by chanchal-sadhukhan**
## Description
Creates:
Create Access Safe Component

## Screenshot
Screenshot from 2022-01-29 12-18-12

### Manual test cases run
_For each manual test case, list the steps to test or reproduce the PR._
- _Description of manual test 1_
- _Description of manual test 2_
- _Description of manual test 3_

## Platform tested
- [x] Large
- [x] Tablet
- [x] Mobile

### CODE REVIEW

File: src/components/primitive/access-safe/access-safe.component.props.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  /**
+   * optional image prop for the item
+   */
+  image?: ImageComponentProps;
+
+  /**
+   * sub heading for the access safe item
+   */
+  subHeading?: TextComponentProps;
```
Comment (dsouzaedison) [inline src/components/primitive/access-safe/access-safe.component.props.ts]: Make it subheader

File: src/components/primitive/access-safe/access-safe.component.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        {/* @ts-ignore */}
+        {previousButton && <CancelButton {...previousButton} />}
+        {!nextButton && (
+          <SuccessButton
+            label={{ text: 'Continue' }}
+            variant="primary"
+            onClick={() => {}}
+          />
+        )}
+        {/* @ts-ignore */}
```
Comment (dsouzaedison) [inline src/components/primitive/access-safe/access-safe.component.tsx:99]: What's the error? Always add the error as comment next to ts-ignore.

**Review by dsouzaedison — CHANGES_REQUESTED** [review]

File: src/components/primitive/access-safe/access-safe.component.tsx (hunk 2 above)
Comment (chanchal-sadhukhan) [reply src/components/primitive/access-safe/access-safe.component.tsx:99]: We've made nextButton and previousButton props optional that's why the error is occurring

Comment (dsouzaedison) [reply src/components/primitive/access-safe/access-safe.component.tsx:99]: Can't you add {...(nextButton ?? {})}

Comment (chanchal-sadhukhan) [reply src/components/primitive/access-safe/access-safe.component.tsx:99]: It's not working

**Review by dsouzaedison — APPROVED** [review]

**Pull request merged**